src/boot/efi/proto/Hash2.h

```diff
@@ -0,0 +1,171 @@
+/** @file
+        EFI_HASH2_SERVICE_BINDING_PROTOCOL as defined in UEFI 2.5.
+        EFI_HASH2_PROTOCOL as defined in UEFI 2.5.
+        The EFI Hash2 Service Binding Protocol is used to locate hashing services support
+        provided by a driver and to create and destroy instances of the EFI Hash2 Protocol
+        so that a multiple drivers can use the underlying hashing services.
+        EFI_HASH2_PROTOCOL describes hashing functions for which the algorithm-required
+        message padding and finalization are performed by the supporting driver.
+
+Copyright (c) 2015, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
SPDX as first line, and also needs to be described in LICENSES/

Reply:
As per previous comments, it will be replaced with custom sha1 implementation, so this file will be removed.